docs/docs/troubleshooting.md

```diff
@@ -0,0 +1,69 @@
+# Troubleshooting
+
+This section contains information about how to debug your Contrast deployment.
+
+## Logging
+
+Collecting logs can be a good first step to identify problems in your
+deployment. Both the CLI and the Contrast Coordinator as well as the Initializer
+can be configured to emit additional logs.
+
+### CLI
+
+The CLI logs can be configured with the `--log-level` command-line flag, which
+can be set to either `debug`, `info`, `warn` or `error`. The default is `info`.
+Setting this to `debug` can get more fine-grained information as to where the
+problem lies.
+
+### Coordinator and Initializer
+
+The logs from the Coordinator and the Initializer can be configured via the
+environment variables `CONTRAST_LOG_LEVEL`, `CONTRAST_LOG_FORMAT` and
+`CONTRAST_LOG_SUBSYSTEMS`.
+- `CONTRAST_LOG_LEVEL` can be set to one of either `debug`, `info`, `warn`, or
+  `error`, similar to the CLI (defaults to `info`).
+- `CONTRAST_LOG_FORMAT` can be set to `text` or `json`, determining the output
+  format (defaults to `text`).
+- `CONTRAST_LOG_SUBSYSTEMS` is a comma-seperated list of subsystems that should
+  be enabled for logging, which are disabled by default. Subsystems include:
+  `snp-issuer`, `kds-getter`, and `snp-validator`. To enable all subsystems, use
+  `*` as the value for this environment variable.
+
+Warnings and error messages from subsystems get printed regardless of whether
+the subsystem is listed in the `CONTRAST_LOG_SUBSYSTEMS` environment variable.
+
+#### Coordinator debug logging
+
+To configure debug logging with all subsystems for your Coordinator, add the
+following to your container definition before running `contrast generate`.
```

Comment:
Notice the user usually won't call generate on the Coordinator YAML. We should setup the coordinator policy in a way that allows setting/changing these env variables without regeneration.

Reply:
Created a backlog item for that.